### PR TITLE
feat: audio input gain control

### LINKED ISSUE #727: Increase audio input gain directly in meyda
**Is your feature request related to a problem? Please describe.**
Increase the gain in meyda, so that the input audio can be controlled.

**Describe the solution you'd like**
Audio Gain component in the bottom bar next to fps. 

**Describe alternatives you've considered**
Increase the volume in the application that is playing back the audio. Having an external audio interface that can increase the gain, but for the price of having distorted audio. I want to keep the original audio input untouched, so that I can still record it.

### PR DESCRIPTION
**Pull request by TimPietrusky**
See #727

### CODE REVIEW

**Comment by TimPietrusky** [comment]
* Set the gain via "Input Device Config"
* Use the gainNode in meyda

File: src/components/InputDeviceConfig/AudioVideo.vue [hunk 1]
```diff
@@ -71,13 +88,17 @@ export default {
   data() {
     return {
       iVTitle: "Media Input Config",
-      iVBody:
-        "Configure your audio and video inputs here. Click Renumerate Devices to scan for new sources.",
+      iVBody: `Configure your audio and video inputs here. Click "Re-scan Devices" to scan for new sources.`,
```
Comment (2xAA) [inline src/components/InputDeviceConfig/AudioVideo.vue:91]: A small copy correction. Has been bugging me for ages 😅

File: src/components/InputDeviceConfig/AudioVideo.vue [hunk 2]
```diff
@@ -117,13 +138,27 @@ export default {
         this.$modV.setupMedia({ videoId: value });
         this.switchingVideo = false;
       }
+    },
+
+    maxGain() {
+      return this.$modV.gainNode.gain.maxValue.toPrecision(1 + 2).split("e")[0];
```
Comment (2xAA) [inline src/components/InputDeviceConfig/AudioVideo.vue:144]: We split off the scientific notation here as it kind of breaks the range slider.

File: src/components/InputDeviceConfig/AudioVideo.vue [hunk 3]
```diff
@@ -117,13 +138,27 @@ export default {
...
+    },
+
+    minGain() {
+      return 0;
```
Comment (2xAA) [inline src/components/InputDeviceConfig/AudioVideo.vue:148]: We don't use the `minValue` from the gain node as it returns a negative value.
Negative gain inverts the signal, which is confusing and not expected for a gain slider.

File: src/components/InputDeviceConfig/AudioVideo.vue [hunk 4]
```diff
@@ -71,13 +88,17 @@ export default {
   data() {
     return {
       iVTitle: "Media Input Config",
-      iVBody:
-        "Configure your audio and video inputs here. Click Renumerate Devices to scan for new sources.",
+      iVBody: `Configure your audio and video inputs here. Click "Re-scan Devices" to scan for new sources.`,
       switchingAudio: false,
-      switchingVideo: false
+      switchingVideo: false,
+      gainRangeValue: 1
```
Comment (2xAA) [inline src/components/InputDeviceConfig/AudioVideo.vue:94]: Using a value on data so we can keep track of what the value is doing.
Vue wasn't aware of the value before this.

We also update the value when the component is created below in the `created` function.

**Comment by 2xAA** [comment]
@TimPietrusky If you're happy with this PR, we can merge!

**Comment by TimPietrusky** [comment]
I will not merge this yet, as it's "hard" to get back to 1 or any specific value, as we can't use a right click on the slider as with the other slider we have. I will use the branch for my gig, but we should refactor this so that it's also possible to set the value via input field. 

Also the tooltip of the range-slider and the value shown on the right are not the same in some cases, which is misleading.

**Comment by 2xAA** [comment]
@TimPietrusky Ready to review again. This should have addressed your comments

**Pull request merged**